ntoskrnl/ke/i386/mproc.c

```diff
@@ -127,7 +128,7 @@ KeStartAllProcessors(VOID)
         KeLoaderBlock->Thread = (ULONG_PTR)&APInfo->Pcr.Prcb->IdleThread;
 
         // Start the CPU
-        DPRINT("Attempting to Start a CPU with number: %u\n", ProcessorCount);
+        DPRINT("Attempting to start CPU %Iu\n", ProcessorCount);
```

Comment:
These DPRINTs are here for a reason, both of the ones you touched, Leave them alone.

This code isn't compiled unless you're using ntkrnlmp, and if you are using ntkrnlmp this is required information until any of this remotely works.

Reply:
Serges suggestion is a clear improvement, that does not destroy any information.
I do support it to 100% from the shortening perspective.
Only thing I am unsure about is the %u -> &Iu part, as I am very unfamiliar with format strings.

Reply:
There's two reasons i don't like this 
1) He's adding extra changes he's not referencing in the PR at all.
Which is something every other dev has commented on now
2) Denoting the DPRINTs is my biggest concern here. 

Leave the DPRINTs alone, they're here for a reason.

Reply:
Which denoting? It was a DPRINT before, and it is a DPRINT now...

Reply:
The other 2 are DPRINT1's

Reply:
Both DPRINT1 that Serge turned to DPRINT will still log, because he commented the #define NDEBUG in that file.
That does fix a compiler warning in rls-configuration about "redefining NDEBUG". It will still log as before.
Imho is ok.

Reply:
I don't think globally undefining NDEBUG for an entire file that's 17+ years old is a good idea either

Reply:
I checked the file: there are no UNIMPLEMENTED* in here, and only 2 DPRINTs. And both are touched by Serge. So his change with the NDEBUG will have no undesired side-effects of enabling any other logging.
But it is more speaking like that: It means: I want to observe for a while what this file does. And as said: it fixes a compile-time warning in rls.
The change should be fine. Really.